packages/ui/src/components/AccountDisplay.tsx

```diff
@@ -73,10 +80,10 @@ const AccountDisplay = ({
   }, [address, api, isApiReady])
 
   return (
-    <AccountDisplayStyled>
+    <div className={className}>
```

Comment:
The problem is that styling of this component is broken right now in prod... e.g we have stuff like this:

![image](https://github.com/ChainSafe/Multix/assets/33178835/0874ef82-e7c7-464e-b8ab-989bbd5c9cdb)

 This component is called with a class in other components to get some specific style. And because the `className` isn't used at all right now, this didn't work. So unless we figure things out and fix it the right way, I revert this and the day we touch it again, I'll make sure to review carefully.